Rubberduck.Parsing/Annotations/FolderAnnotation.cs

```diff
@@ -13,11 +13,7 @@ public sealed class FolderAnnotation : AnnotationBase
             IEnumerable<string> parameters)
             : base(AnnotationType.Folder, qualifiedSelection)
         {
-            if (parameters.Count() != 1)
-            {
-                throw new InvalidAnnotationArgumentException(string.Format("{0} expects exactly one argument, the folder, but none or more than one were passed.", this.GetType().Name));
-            }
-            _folderName = parameters.First();
+            _folderName = parameters.FirstOrDefault() ?? string.Empty;
```

Comment:
Basically, we don't want Rubberduck throwing or complaining (except in this inspection) in any way about bad annotations--it should just ignore them.